docs/examples/koa-webhook-bot.js

```diff
@@ -23,8 +27,12 @@ bot.telegram.setWebhook('https://-----.localtunnel.me/secret-path')
 const app = new Koa()
 app.use(koaBody())
 // @ts-ignore
-app.use((ctx, next) => ctx.method === 'POST' || ctx.url === '/secret-path'
-  ? bot.handleUpdate(ctx.request.body)
-  : next()
-)
+app.use((ctx, next) => {
+  if (ctx.url === '/secret-path') {
+    bot.handleUpdate(ctx.request.body)
```

Comment:
```suggestion
    await bot.handleUpdate(ctx.request.body)
```

Reply:
it the final exprssion before return. so we can ignore `await`

Reply:
It affects backpressure and error handling.

Reply:
ok. i can fix it now